doma-kotlin/src/main/kotlin/org/seasar/doma/kotlin/jdbc/criteria/declaration/KJoinDeclaration.kt

```diff
@@ -10,47 +10,47 @@ import org.seasar.doma.jdbc.criteria.tuple.Tuple2
 import org.seasar.doma.jdbc.criteria.tuple.Tuple3
 
 class KJoinDeclaration(private val declaration: JoinDeclaration) {
-    fun <PROPERTY : Any> eq(left: PropertyMetamodel<PROPERTY>, right: PROPERTY) {
+    fun <PROPERTY : Any> eq(left: PropertyMetamodel<PROPERTY>, right: PROPERTY?) {
```

Comment:
https://github.com/domaframework/doma/blob/e5a7271e19054d04fc92889f35dc966327d70cc2/doma-core/src/main/java/org/seasar/doma/jdbc/criteria/declaration/ComparisonDeclaration.java#L40-L45

in java-code, allow null.